content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+
+Once the order is placed, the tags for the items just purchased in the :guilabel:`Your Order` field
+change from orange :guilabel:`To Order` tags to red :guilabel:`Ordered` tags.
+
+Track an order
+--------------
+
+When orders have been sent to the vendors, the tags for the items in the :guilabel:`Your Order`
+summary change from red :guilabel:`Ordered` tags to blue :guilabel:`Sent` tags.
+
+Once orders have been received and verified, the tags change from blue :guilabel:`Sent` tags to
+green :guilabel:`Received` tags.
+
+Receive an order
+----------------
+
+When orders are received at the delivery location, they are confirmed, and a notification is sent
+to the employee who ordered the food.
+
+My Orders
+=========
+
+To view a full list of all orders placed in the *Lunch* app for the currently signed-in user,
+navigate to :menuselection:`Lunch app --> My Lunch: My Order History`. This navigates to the
+:guilabel:`My Orders` dashboard. The data is filtered by :guilabel:`My Orders` and grouped by
+:guilabel:`Order Date: Day`, by default.
+
+All products appear in a list view, organized by date. The list displays the :guilabel:`Order Date`,
+:guilabel:`Vendor`, :guilabel:`Product`, :guilabel:`Extras`, :guilabel:`Notes`, :guilabel:`User`,
+:guilabel:`Lunch Location`, :guilabel:`Price`, :guilabel:`Status`, and :guilabel:`Company`
+information.
+
+The total cost for each order is displayed on the line containing the order date. At the bottom of
+the list, the total amount paid for all the orders appears.
+
+At the end of each product with a status of :guilabel:`Ordered`, a :guilabel:`X Cancel` button
+appears. Click :guilabel:`X Cancel` to cancel that product order. Once a product order has been
+canceled, the money paid for that product is refunded and appears in the user's account.
+
+At the end of each item with a status of :guilabel:`Received`, a :guilabel:`Re-order` button
+appears. Click :guilabel:`Re-order` to instantly reorder the product. The new order appears in the
+list, under the current date, and the product is paid for, and the money is deducted from the user's
+account.
+
+.. image:: orders/my-orders.png
+   :align: center
+   :alt: The list view that appears when navigating to the My Orders dashboard.
+
+My Account
+==========
+
+To view a summary of all transactions in the user's account, navigate to :menuselection:`Lunch app
+--> My Lunch: My Account History`. This navigates to the :guilabel:`My Account` dashboard.
+
+The default presentation displays all entries from newest to oldest. The :guilabel:`Date`,
+:guilabel:`Description`, and :guilabel:`Amount` are the only fields displayed in the list.
+
+Entries with a negative :guilabel:`Amount` listed are typically for products purchased in the
+*Lunch* app. Entries with a positive balance are typically either from adding funds to the user's
+lunch account, or for canceled orders that were refunded to the user.
```

Comment:
might be good to write out how positive and negative balance appear: '-' '+' 

as in Odoo '()' can also mean negative